Formula/patat.rb

```diff
@@ -0,0 +1,58 @@
+class Patat < Formula
+  desc "Terminal-based presentations using Pandoc"
```

Comment:
Shouldn't it depend on pandoc then?

Reply:
It uses Pandoc as a Haskell library, so it's essentially compiled into the executable.